report_page_orientation/demo/report_page_orientation_demo.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="report_page_orientation_demo_reports" model="ir.actions.report">
+        <field name="name">Demo report</field>
+        <field name="model">res.users</field>
+        <field name="report_type">qweb-pdf</field>
+        <field
+            name="report_name"
+        >report_page_orientation.report_page_orientation_demo_report_view</field>
+        <field
+            name="report_file"
+        >report_page_orientation.report_page_orientation_demo_report_view</field>
+        <field name="paperformat_id" ref="base.paperformat_euro" />
+        <field name="binding_model_id" ref="base.model_res_users" />
+        <field name="attachment">'Demo.pdf'</field>
+        <field name="binding_type">report</field>
+    </record>
+    <template id="report_page_orientation_demo_report_view">
+        <t t-call="web.html_container">
+            <t t-call="web.external_layout">
+                <div class="page">
+                    List of Current Users in The system
+                    <ul>
+                        <li t-foreach="docs" t-as="doc">
+                            <t t-esc="doc.name" />
+                        </li>
+                    </ul>
+                    <h2>Hello this is a page</h2>
+                    <t t-foreach="range(100)" t-as="index">
```

Comment:
@tom this does one portrait page (1xp), 100 rotated pages (100 x r) and closes with a portrait page (1x p)  -  the 100 number seems unnecessary.
wouldn't it be better to have test data that checks different patterns? ( different alternations, with different first pages and last pages, to uncover corner cases) 
1x p - 3 x r - 1x r
1x r - 3x p - 2 x r 


the test infrastructure is there should be easy to adapt.